modules/10-basics/15-prefix-notation/description.en.yml

```diff
@@ -1,8 +1,8 @@
 ---
 
-name: Префиксная нотация
+name: Prefix notation
 theory: |
-  Как вы могли заметить, в популярных языках программирования, вызов функции выглядит примерно так `f(arg1, arg2, arg3)`, однако в Clojure, как и в других лиспо-подобных языках, используется префиксная нотация (польская запись). С непривычки такой способ записи может показаться неудобным, однако при вложенном вызове становится очевидным удобство такого подхода (опять же, потребуется некоторое время, чтобы привыкнуть). Например, вложенный вызов функции сложения, вычитания и умножения (да, математические операции тоже функции!) выглядит следующим образом:
+  As you may have noticed, in popular programming languages, a function call looks something like `f(arg1, arg2, arg3)`, but Clojure, like other Lisp-like languages, uses prefix notation (Polish notation). This way of notation may seem awkward at first sight, but with a nested call, the convenience of this approach becomes apparent (it will take some time to get used to it). For example, a nested function call for addition, subtraction and multiplication (yes, mathematical operations are functions too!) looks like this:
```

Comment:
This way of notation => This notation style
function call for addition => function call for **an** addition